### PR TITLE
Task

### PR DESCRIPTION
**Pull request by 88mary256**

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/pages/Tasks.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    /**
+     * This method search a task and delete a it.
+     * @param text string.
+     */
+    public void deleteTask(final String text) {
+        for (final WebElement element : this.tasksNames) {
+            if (element.getText().equals(text)) {
+                getParent(getParent(getParent(element)))
```
Comment (d4vidmc) [inline src/main/java/org/fundacionjala/pivotal/pages/Tasks.java:80]: Not sure why you need to iterate through parents elements when each element on page can be located using locators properly.

File: src/main/java/org/fundacionjala/pivotal/pages/Tasks.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        }
+    }
+
+    /**
+     * This method check if exist a task with text.
+     * @param text string
+     * @return boolean
+     **/
+    public boolean existTask(final String text) {
+        for (final WebElement element : this.tasksNames) {
```
Comment (d4vidmc) [inline src/main/java/org/fundacionjala/pivotal/pages/Tasks.java:94]: You are making an iteration over a WebElement to look for a text match, there is an option to check if a specific text is listed.
you should see .isDisplayed() with a try catch exception

**Review by d4vidmc — DISMISSED** [review]
You may change keyword exist to display to show you are running GUI Automation test rather than API testing.
doesn't exist to is not displayed

File: src/main/java/org/fundacionjala/pivotal/pages/Tasks.java (hunk 2 above)
Comment (88mary256) [reply src/main/java/org/fundacionjala/pivotal/pages/Tasks.java:94]: What is the specific option?, could you pleas attach an example?

File: src/main/java/org/fundacionjala/pivotal/pages/Tasks.java (hunk 1 above)
Comment (88mary256) [reply src/main/java/org/fundacionjala/pivotal/pages/Tasks.java:80]: check the tasks, all task have same button to delete, so to get the correct button you need get the button in the same container than the text

File: src/main/java/org/fundacionjala/pivotal/pages/Tasks.java (hunk 2 above)
Comment (d4vidmc) [reply src/main/java/org/fundacionjala/pivotal/pages/Tasks.java:94]: driver.findElement(<WebElement>).isDisplayed();

**Review by rrchcr8 — APPROVED** [review]
it works

**Pull request merged**